prac_06/guitar.py

```diff
@@ -0,0 +1,73 @@
+"""CP1404 Practical - Guitar Class & Program
+
+Guitar Class
+Estimate: 30min
+Actual: 25min
+"""
+
+
+class Guitar:
+    """Represent a Guitar Class object."""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise a Guitar instance.
+        """
+        self.name = name
+        self.year = year
+        self.cost = cost
+        self.age = self.get_age()
+
+    def __str__(self):
+        """Returns a string describing the Guitar Object"""
+        return f"{self.name}({self.year}): ${self.cost}"
+
+    def get_age(self):
+        """Determines Guitars Age"""
+        age = 2023 - self.year
+        return age
+
+    def is_vintage(self):
+        """Determines whether the Guitar is Vintage or not"""
+        if self.age >= 50:
```

Comment:
You can move the return to the top line and remove everything below. Having the else return vintage = False is redundant in this case since if it doesn't meet the condition to be True, it'll always be False.